docs/content/guides/author-apps/kubernetes/howto-yaml-base-support/index.md

```diff
@@ -0,0 +1,101 @@
+---
+type: docs
+title: "How-To: Apply property overrides on Kubernetes resources using Radius"
+linkTitle: "Override properties using base YAML"
+description: "Learn how to apply property overrides on Kubernetes resources using Radius and existing Kubernetes YAML manifests."
+weight: 500
+categories: "How-To"
+tags: ["containers","Kubernetes"]
+---
+
+This guide will provide an overview of how to: 
+
+- Use Radius to apply property overrides on top of a given Kubernetes resource using its base [Kubernetes YAML](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#writing-a-deployment-spec) manifest.
+- Migrate your existing Kubernetes resource to Radius using its base [Kubernetes YAML](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#writing-a-deployment-spec) manifest.
+
+## Prerequisites
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [rad CLI]({{< ref getting-started >}})
+- [Radius initialized with `rad init`]({{< ref howto-environment >}})
+
+## Step 1: Define a base Kubernetes YAML file
+
+Begin by creating a file named `manifest.yaml` which defines a Kubernetes application named `base-yaml-app` that includes a Deployment with a single container named `log-collector`:
+
+{{< rad file="snippets/manifest.yaml" embed=true lang="yaml">}}
+
+## Step 2: Define a Radius application
+
+Create another file named `app.bicep` which defines a Radius application named `base-yaml-app` along with a container named `base-yaml-app`. Additionally, include a `runtimes` property that references the `manifest.yaml` file as the base Kubernetes YAML manifest on top of which Radius will apply the properties defined in `app.bicep`.
+
+{{< rad file="snippets/app.bicep" embed=true >}}
+
+> **Note**: the names of the Kubernetes Deployment and the Radius container match each other. For more information on namespace and resource name matching, refer to the [containers overview guide]({{< ref "guides/author-apps/containers/overview#base-kubernetes-yaml" >}}).
+
+## Step 3: Deploy the application and containers
+
+Ensure that your `manifest.yaml` and `app.bicep` files are in the same directory, then deploy your application:
+
+```bash
+rad deploy app.bicep
+```
+
+> **Note**: in the deployment, Radius will override any parameters in the base Kubernetes YAML manifest (i.e. `manifest.yaml` file) with any values specified in the Radius application definition (i.e. `app.bicep` file) for the same parameters.
```

Comment:
@youngbupark - please review this for accuracy

Reply:
Yes it is right. Can you please add the `Limitation` section which describe supported kubernetes resources ?

You can find the details in the following code. 

https://github.com/radius-project/radius/blob/c262df7aee22cf1ebea2fa66d3e70cfdaa135ca0/pkg/corerp/frontend/controller/containers/validator.go#L114-L119

Reply:
@youngbupark 

Thanks, the limitations are already live in the docs: https://docs.radapp.io/guides/author-apps/containers/overview/#base-kubernetes-yaml

I also included a reference to it in this page: https://github.com/radius-project/docs/pull/1024/files#diff-2db3e8ae3a31ec4507e63e2e0b2453d1d2f0b6dc2205502e1487e964b925ef54R34